style.css

```diff
@@ -0,0 +1,457 @@
+* {
+	box-sizing: border-box;
+}
+
+
+body {
+  margin: 0;
+  background: #aabbfb;
+  font-family: "Source Sans Pro";
+  font-size: 16px;
+}
+
+.nav-drop {
+	display: none;
+}
+
+.dropbtn {
+	background-color: #fbeaaa;
+	color:  black;
+	padding: 5.5px;
+	font-size: 1.8rem;
+	border: 0.5px solid;
+	padding-right: 20px;
```

Comment:
padding: 6.5px 20px 0 20px; would be a quicker way to write you padding code on one line